tests/unittest/serve/test_cluster_manager_worker.py

```
@@ -0,0 +1,211 @@
+    }
+
+    assert await cluster_manager.is_ready() == True
+    await cluster_manager.stop()
+
+
+@pytest.mark.parametrize("config", ["http", "etcd"], indirect=True)
+@pytest.mark.threadleak(enabled=False)
+@pytest.mark.timeout(20)
+@pytest_async_module
+async def test_cluster_manager(cluster_manager, storage_client, config):
+    cluster_manager.current_ctx_worker_num == 0
```

Ruff (0.13.1)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)